Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Commands/CancelDeletionProcess/HandlerTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Backbone.BuildingBlocks.Application.Abstractions.Exceptions;
+using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.UserContext;
+using Backbone.Modules.Devices.Application.Identities.Commands.CancelDeletionProcess;
+using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Tooling;
+using FakeItEasy;
+using FluentAssertions;
+using Xunit;
+using static Backbone.UnitTestTools.Data.TestDataGenerator;
+
+namespace Backbone.Modules.Devices.Application.Tests.Tests.Identities.Commands.CancelDeletionProcess;
+public class HandlerTests
+{
+    [Fact]
+    public async Task Happy_path()
+    {
+        // Arrange
+        var activeIdentity = TestDataGenerator.CreateIdentityWithApprovedDeletionProcess(SystemTime.UtcNow.Date);
+        var activeDevice = activeIdentity.Devices[0];
+
+        var mockIdentitiesRepository = A.Fake<IIdentitiesRepository>();
```

Comment:
This is not used as a mock and therefore should not be called a mock.

BUT: your test is missing an assertion that makes sure the Identity and the DeletionProcess were updated in the database